### PR TITLE
Add status: fields

### PR DESCRIPTION
**Pull request by aemengo**
Add status fields _per_ [0094-add-status-field.md](https://github.com/buildpacks/rfcs/blob/main/text/0094-add-status-field.md)

### CODE REVIEW

**Review by hone — CHANGES_REQUESTED** [review]
@aemengo thanks for doing this. I reviewed this and it looks good. I've noticed that RFC 35 is missing. Can you add that in this PR.

I also noticed that I repeated RFC 84 (the numbering), but I'll fix that in a separate PR.

**Review by nebhale — APPROVED** [review]

**Review by hone — APPROVED** [review]

File: text/0079-create-stackify-repo.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - Name: Create stackify repo
 - Start Date: 2020-11-06
 - Authors: @martyspiewak @mdelillo @dumez-k
+- Status: Approved
```
Comment (ekcasey) [inline text/0079-create-stackify-repo.md:6]:
```suggestion
- Status: Superseded
```

I am assuming this is superseded by [RFC 0096](https://github.com/buildpacks/rfcs/blob/main/text/0096-remove-stacks-mixins.md)?

cc @sclevine

File: text/0061-relax-mixin-contract.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 - Name: Relax Mixin Contract
 - Start Date: 2020-08-12
 - Author(s): @sclevine
+- Status: Approved
```
Comment (ekcasey) [inline text/0061-relax-mixin-contract.md:6]:
```suggestion
- Status: Superseded
```
Is this superseded by RFC 0096?

File: text/0079-create-stackify-repo.md (hunk 1 above)
Comment (aemengo) [reply text/0079-create-stackify-repo.md]: I apologize for missing this.

_Addressed in latest commit._

File: text/0061-relax-mixin-contract.md (hunk 2 above)
Comment (aemengo) [reply text/0061-relax-mixin-contract.md]: I apologize for missing this.

_Addressed in latest commit._

**Review by ekcasey — APPROVED** [review]

**Review by sclevine — APPROVED** [review]

**Pull request merged**